packages/buidler-core/src/internal/buidler-evm/stack-traces/compiler-types.ts

```diff
@@ -49,4 +49,7 @@ export interface CompilerOutputBytecode {
       [libraryName: string]: Array<{ start: 0; length: 20 }>;
     };
   };
+  immutableReferences?: {
```

Comment:
I didn't know about this! 🥳 This is great news! Will make everything easier.

I compiled a few contracts and couldn't find it 😅